ext/project_data.yaml

```diff
@@ -13,14 +13,6 @@ gem_require_path: 'lib'
 gem_test_files: 'spec/**/*'
 gem_executables: 'hiera'
 gem_default_executables: 'hiera'
-gem_platform_dependencies:
-  x86-mingw32:
-    gem_runtime_dependencies:
-      win32console: '1.3.2'
-      win32-dir: '~> 0.4.8'
-  x64-mingw32:
-    gem_runtime_dependencies:
-      win32-dir: '~> 0.4.8'
 bundle_platforms:
   x86-mingw32: mingw
   x64-mingw32: x64_mingw
```

Comment:
We can nuke `bundle_platforms` too. It's only used if we have a non-empty `gem_platform_dependencies` hash: https://github.com/puppetlabs/hiera/pull/393/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478R25